core/src/main/scala/org/apache/spark/scheduler/Stage.scala

```diff
@@ -118,7 +119,7 @@ private[scheduler] abstract class Stage(
    */
   private[scheduler] def failedOnFetchAndShouldAbort(stageAttemptId: Int): Boolean = {
     fetchFailedAttemptIds.add(stageAttemptId)
-    fetchFailedAttemptIds.size >= Stage.MAX_CONSECUTIVE_FETCH_FAILURES
+    fetchFailedAttemptIds.size >= maxConsecutiveFetchFailure
```

Comment:
not a big deal, but since `maxConsecutiveFetchFailure` isn't really changing from stage to stage, so I think rather than making it a member variable of stage, maybe its makes more sense to make it a parameter to this method. thoughts?